<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetProperties</class>
 <widget class="QWidget" name="TargetProperties">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="4" column="0">
      <widget class="QLabel" name="labelCullMode">
       <property name="text">
        <string>Cull Face</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="DataComboBox" name="cullMode"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelFrontFace">
       <property name="text">
        <string>Front Face</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="DataComboBox" name="frontFace"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelLogicOperation">
       <property name="text">
        <string>Logic Operation</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="DataComboBox" name="logicOperation"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelBlendConstant">
       <property name="text">
        <string>Blend Constant</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="ColorPicker" name="blendConstant"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelWidth">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="width">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelHeight">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelLayers">
       <property name="text">
        <string>Layers</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSamples">
       <property name="text">
        <string>Samples</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="height">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="layers">
       <property name="maximum">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="samples">
       <property name="maximum">
        <number>256</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataComboBox</class>
   <extends>QComboBox</extends>
   <header>session/DataComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QCheckBox</extends>
   <header>session/ColorPicker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
